lib/xapi-stdext-date/date.ml

```diff
@@ -12,19 +12,9 @@
  * GNU Lesser General Public License for more details.
  *)
 
-type iso8601 = string
+(* ==== RFC822 ==== *)
```

Comment:
You might want to use `(**` for these comments to make them appear in the docs

Reply:
this was more meant to help separate functions related to this rfc822 type, and the iso8601 type whilst looking at the file